src/redux/configureStore.js

```diff
@@ -0,0 +1,9 @@
+import { createStore, combineReducers } from 'react-redux';
```

Comment:
- [ ] You are required to import `createStore` and `combineReducers` from **redux**, not **react-redux**. Kindly fix that.